src/Cli/dotnet/Installer/Windows/WorkloadPackRecord.cs

```diff
@@ -0,0 +1,61 @@
+﻿// Copyright (c) .NET Foundation and contributors. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+#nullable disable
+
+using System;
+using NuGet.Versioning;
+
+namespace Microsoft.DotNet.Installer.Windows
+{
+    /// <summary>
+    /// Represents a single workload pack installation record in the registry created
+    /// by a workload pack MSI.
+    /// </summary>
+    internal class WorkloadPackRecord
```

Comment:
Can we specify in the name that this is for the msi/ registry install, something line WorkloadPackRegistryRecord?